src/main/java/com/github/mkopylec/sessioncouchbase/persistent/CouchbaseSession.java

```diff
@@ -60,12 +60,12 @@ public static String globalAttributeName(String attributeName) {
 
     @Override
     public long getCreationTime() {
-        return round((double) globalAttributes.get(CREATION_TIME_ATTRIBUTE));
+        return round(new Double(globalAttributes.get(CREATION_TIME_ATTRIBUTE).toString()));
```

Comment:
Done, I'm using Couchbase 4.1.0, specifically this package (community version) http://packages.couchbase.com/releases/4.1.0/couchbase-server-community_4.1.0-ubuntu12.04_amd64.deb . I'll try to provide a test environment when I get some time